builtin/game/features.lua

```diff
@@ -14,7 +14,7 @@ core.features = {
 	object_independent_selectionbox = true,
 	httpfetch_binary_data = true,
 	formspec_version_element = true,
-	area_store_persistent_ids = true,
+	area_store_persistent_ids = true
```

Comment:
Same here, the comma would have to be added back if another entry is added so don't remove it.